sdk/database/dbplugin/v5/proto/database.proto

```diff
@@ -27,6 +27,8 @@ message NewUserRequest {
     google.protobuf.Timestamp expiration = 3;
     Statements statements = 4;
     Statements rollback_statements = 5;
+    int32 credential_type = 6;
+    bytes public_key = 7;
```

Comment:
Maybe a minor thing, but since we're only using PEM-encoded keys, those should all be valid `string`s, right?

I mostly worry that `bytes` (or `[]byte`) above might lead someone to think we're using something like DER.

Reply:
Yep, I originally used a `string` type here. I would be fine going back to `string` since we are going with PEM encoding. Will do so if others agree.

Reply:
I don't have a strong preference either way. FWIW the standard library call `pem.EncodeToMemory` also returns it as `[]byte`.